docs/UserGuide.md

```diff
@@ -29,9 +29,12 @@ Workbook is optimized for fast typists and utilizes a **Command Line Interface (
 
 3. Copy the file to a folder of your choice to store data from your WorkBook.
 
-4. Using your command terminal, navigate to the folder where you placed your WorkBook at and fire it up by inputting: `java -jar WorkBook.jar` in your terminal. <br> The GUI with sample data as shown below should appear in a few seconds.<br>
-   ![Ui](images/Ui.png)
+4. Using your command terminal, navigate to the folder where you placed your WorkBook at and fire it up by inputting: `java -jar WorkBook.jar` in your terminal. <br> The GUI with sample data similar to one of the images below should appear in a few seconds.<br>
```

Comment:
possible to put the narrow ui on top and resize the png so it actually looks like its the smaller window?
```suggestion
4. Using your command terminal, navigate to the folder where you placed your WorkBook at and fire it up by inputting: `java -jar WorkBook.jar` in your terminal. <br> The GUI with sample data should appear in a few seconds. Depending on the window size, you will see one of the two layouts shown below.<br>
```



![Screenshot 2022-11-02 at 1 04 38 AM](https://user-images.githubusercontent.com/77665182/199294615-8492cdc5-7764-4cbc-ba90-137bb8c714d0.png)